python/cuml/preprocessing/model_selection.py

```diff
@@ -20,12 +20,12 @@
 
 
 @jit
-def _shuffle_idx(df: cudf.DataFrame) -> np.ndarray:
+def _shuffle_idx(idx: np.ndarray) -> np.ndarray:
```

Comment:
This now shuffles in-place right? Should update type signature and docs to align with that.